crates/usvg-parser/src/style.rs

```diff
@@ -187,6 +187,8 @@ fn convert_paint(
                 from_fallback(node, fallback, opacity)
             }
         }
+        // Ignore `context-fill` and `context-stroke for now
```

Comment:
???

Reply:
Since those two are part of `svgtypes` I have to include them in the pattern matching for it to compile, but since the PR for that isn't finished yet, I have to disregard it for now. Should I do something else instead?

Reply:
Oh, my bad. I forgot we added them to `svgtypes` already.